src/languageProvider/locales/en_US.json

```diff
@@ -80,15 +87,26 @@
   "sort.asc": "Ascending",
   "sort.default": "Sort",
   "sort.desc": "Descending",
-  "str.RAISE": "RAISED",
+  "str.bot": "BOT",
   "str.end": "Ended",
   "str.outcome": "OUTCOMES",
+  "str.qtum": "QTUM",
   "str.raise": "Raised",
+  "str.RAISE": "RAISED",
+  "str.sort": "Sort",
   "topbar.betting": "Betting",
   "topbar.completed": "Completed",
   "topbar.create": "Create an Event",
   "topbar.event": "Event",
   "topbar.events": "Events",
   "topbar.setting": "Setting",
-  "topbar.voting": "Voting"
+  "topbar.voting": "Voting",
```

Comment:
please add a TODO for chinese translation

Reply:
Henry's translation script will copy the English version to the Chinese version (if missing) so we know which ones still need translating.